gamedata/myz/myz-weapons-catalog.en.csv

```diff
@@ -48,7 +48,7 @@ wmyz-scrapderringer;1;1;1;1;true;Light;Jury-Rigged;myz
 wmyz-scrapflamethrower;2;1;2;1;true;Fire|Heavy;Jury-Rigged;myz
 wmyz-scrapknife;1;1;2;0;;Light;Jury-Rigged;myz
 wmyz-scrapmachete;1;2;2;0;;;Jury-Rigged;myz
-wmyz-scrappistol;1;2;2;2;true;;Jury-Rigged;myz
+wmyz-scrappistol;1;1;2;2;true;;Jury-Rigged;myz
```

Comment:
```suggestion
wmyz-scrappistol;1;2;2;2;true;;Jury-Rigged;myz
```
Are you sure of this change?
In version 4 it is 2 damage for the scrap pistol.

Reply:
I made this change based on the current german PDF versions of the core book and mechatron, which usually already contain errata from english versions, and the free starter booklet from drivethrurpg. My google fu couldn't find anything about this. And the german Mechatron book is from 2020.

Reply:
Oh, i just saw this. I didn't change the damage, that's two in my versions, too. I changed the bonus, which says +1 in my books.

Reply:
> Scrap Pistol: +1 Gear Die, 2 damage
This is the stats for the Scrap Pistol in the v4 of the English PDF (2 dec 2018).
Is it the same for the German version ?

Reply:
Yes, it's the same. +1 Bonus (gear die), 2 Damage. In your proposed version it's +2 Bonus and 2 Damage.

Reply:
Oh yes! I forgot that the 2nd value is for the `grip`.